src/overlays/actors/ovl_Bg_Spot01_Idohashira/z_bg_spot01_idohashira.c

```diff
@@ -9,7 +17,27 @@ void BgSpot01Idohashira_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot01Idohashira_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot01Idohashira_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_808AB504(BgSpot01Idohashira* this, GlobalContext* globalCtx);
+void func_808AB510(BgSpot01Idohashira* this, GlobalContext* globalCtx);
+void func_808AB530(BgSpot01Idohashira* this, GlobalContext* globalCtx);
+void func_808AB570(BgSpot01Idohashira* this, GlobalContext* globalCtx);
+void func_808AB700(BgSpot01Idohashira* this, GlobalContext* globalCtx);
+
+BgSpot01IdohashiraActionFunc D_808AB830[] = {
+    func_808AB504,
+    func_808AB510,
+    func_808AB530,
+    func_808AB570,
+};
+
+InitChainEntry D_808AB840[] = {
+    ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
```

Comment:
sInitChain
the script that generates this will name this for you btw, you dont need to revert it back

Reply:
Ah yeah, I named it that manually since I hadn't decompiled the function yet and didn't want to have to change any .s files.